test/addons_test.go

```diff
@@ -241,6 +243,18 @@ func kafkaFilters(addon v1beta1.AddonInterface) {
 	}
 }
 
+func sparkFilters(addon v1beta1.AddonInterface) {
```

Comment:
what is motivating this removal?

Reply:
When metrics reporting is enabled in Spark Operator, it requires `prometheus-operator` to be installed, more specifically, it depends on `ServiceMonitor` CRD. So we skipping this requirement until there is a way to automatically install addon's dependencies as a part of the testing flow.